projects/wallet-template/src/background/storage.ts

```diff
@@ -4,6 +4,10 @@ const STORAGE_PREFIX = "wallet-template/"
 
 type StorageConfig = {
   password: KeystoreV4
+  keysets: {
+    [key: string]: any
+  }
+  primaryKeysetName: string
```

Comment:
let's default to the first key as primary and remove this key

Reply:
you still need to store something though. if the user selects an account, then closes the browser, and then re-opens it, the extension should remain on the same account.

Reply:
we can turn keyset into a typed union when we add plain accounts and then have plain accounts as a different variant. then handle each variant when decoded.